src/lib/firebase.ts

```diff
@@ -56,6 +56,16 @@ export class Firebase {
     return `/${parts.join("/")}/`;
   }
 
+  // Only used for testing in demo DB. Calls to this function should never be committed to master.
```

Comment:
Take a look at my recent commit - I added a constant at the top of the file that can be set during testing, but should stay `undefined` on production. I mostly implemented a URL parameter, but then I realized that inadvertently visiting the wrong URL could pollute the database. I visit `localhost:8080/?demo` pretty frequently and it doesn't seem like I should mess up the live database if I do that by accident.

The real solution to this and the problem @sfentress brought up this morning would be to have our own local databases, right? This would add other challenges, like serving our own branches for PIs to test. Maybe worth discussing further

Reply:
Should be covered by new PR